admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
```

Comment:
I saw the discussion in the previous PR about what to call these; I'd rather they just simply be referred to as "node components", to align with similar wording elsewhere in the docs.